src/features/classification/applied-by-ai-classification-reason/__tests__/AppliedByAiClassificationReason.test.tsx

```diff
@@ -111,4 +111,67 @@ describe('features/classification/applied-by-ai-classification-reason/AppliedByA
 
         expect(reasonText).toBeVisible();
     });
+
+    test('should render title and citations only when answer is not provided', () => {
+        const expectedCitations = [
+            {
+                content: 'file content for citation',
+                fileId: 'fileId',
+                location: 'cited location',
+                title: 'file title',
+            },
+        ];
+
+        const { container } = renderComponent({
+            answer: undefined,
+            citations: expectedCitations,
+        });
+
+        const boxAiIcon = screen.getByTestId('box-ai-icon');
+        const appliedByWithDate = screen.getByRole('heading', {
+            level: 3,
+            name: messages.appliedByBoxAiOnDate.defaultMessage.replace('{modifiedAt}', modifiedAtDisplayDate),
+        });
```

Comment:
we try to use the literal string value instead of searching for the value through the `messages` object:
```suggestion
            name: "Box AI on ...",
        });
```

Reply:
as well as using `replace(...)` inside of a test is a little sketch since it starts adding logic to the test

Reply:
aaah this is a habitual pattern I borrowed from our other repos, where we prefer the `messages` object vs literal string value. but noted thank you!